src/StdString.h

```diff
@@ -252,7 +252,7 @@ inline int ssicmp(const CT* pA1, const CT* pA2)
 
 inline void sslwr(char *pT, size_t nLen)
 {
-	MakeLower( pT, nLen );
+	//MakeLower( pT, nLen );
```

Comment:
This was the only thing that caused an error when I tried to compile.  Commenting out other things would have been more than the minimum change.  Since StdString is completely unrelated to NewField, keeping changes in StdString to an absolute minimum was more important than consistency.